src/components/DataTable/_data-table.scss

```diff
@@ -1 +1,67 @@
 @import '~carbon-components/scss/components/data-table/data-table';
+
+.table-header-label {
+  &-start {
+    .bx--table-header-label {
+      width: 100%;
+      text-align: start;
+    }
+  }
+
+  &-center {
+    .bx--table-header-label {
+      width: 100%;
+      text-align: center;
+    }
+  }
+
+  &-end {
+    .bx--table-header-label {
+      width: 100%;
+      text-align: end;
+    }
+  }
+}
+
+.bx--data-table td:last-of-type {
```

Comment:
`bx` needs to be replaced with the prefix variable

Reply:
Ok replaced with #{$prefix}